README.md

```diff
@@ -1 +1,150 @@
-# portfolio-mobile-final
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>Portfolio: finish mobile version</b></h3>
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 Portfolio: finish mobile version <a name="about-project"></a>
+
+Portfolio: finish mobile version  project is the second part of my portiflio .
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML</summary>
+  <ul>
+    <li><a href="NA">NA</a></li>
+  </ul>
+</details>
+<details>
+  <summary>CSS</summary>
+  <ul>
+    <li><a href="NA">NA</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- linter setup
+- HTML and CSS file creation
+- creating the work and about me sections of my mobile version portiflio
+<p align="right">
+(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link]()
+
+  > None
+
+  <p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+  ## 💻 Getting Started <a name="getting-started"></a>
+
+  To get a local copy up and running it in your machine, follow the next steps.
+
+  ### Prerequisites
+
+  In order to run this project you need:
+
+- Installation of Live Server plugging code editor.
+- A working browser.
+- A code editor.
+- Installation of NPM.
+- Installation of Git.
+  
+### Setup
+
+Copy the projects URL from github then clone from your code editor
+
+### Install
+
+Install almost all dependencies.
+This one will install stylelinter:
+
+> npm install --save-dev
+> stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+
+### Usage
+
+To run the project, execute in live server.
+
```

Comment:
- [ ] Please provide an easy step-by-step description, explaining how would a Non-technical reader can use this project, it will make it more accessible to different visitors.